docs/source/inthewild.rst

```diff
@@ -59,3 +59,8 @@ Just add a section below by raising a PR on GitHub by
   SQLFluff across the organization, the clarity of our SQL code has risen
   dramatically, freeing up review time to focus on deeper data and
   process-specific questions.
+- `Symend <https://www.symend.com>`_ has a microservices platform supporting our SaaS product.
+  We use SQLFLuff in the CI/CD process of several of our data-oriented microservices.
+  Among other things, it validates our database migration scripts, deployed using
+  `schemachange <https://github.com/Snowflake-Labs/schemachange>`_ and we have near-term plans to
+  implement it for our `dbt <http://www.getdbt.com>`_ projects.
```

Comment:
Need to make these lines shorter and also currently duplicating the dbt link from a previous entry:

```suggestion
- `Symend <https://www.symend.com>`_ has a microservices platform supporting
  our SaaS product. We use SQLFLuff in the CI/CD process of several of our
  data-oriented microservices. Among other things, it validates our database
  migration scripts, deployed using
  `schemachange <https://github.com/Snowflake-Labs/schemachange>`_ and we have
  near-term plans to implement it for our `dbt`_ projects.
```